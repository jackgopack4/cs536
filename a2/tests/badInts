99999999999999999999999999999
2147483648
