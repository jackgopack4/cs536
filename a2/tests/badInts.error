1:1 ***WARNING*** integer literal too large; using max value
2:1 ***WARNING*** integer literal too large; using max value
