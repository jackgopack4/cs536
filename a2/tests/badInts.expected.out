Token: INTLITERAL
LineNum: 1
CharNum: 1
Value: 2147483647

Token: INTLITERAL
LineNum: 2
CharNum: 1
Value: 2147483647

